views/js/pciCreator/dev/mathEntryInteraction/runtime/mathEntryInteraction.js

```diff
@@ -541,20 +549,29 @@ define([
                     equal:  { label: '=',           latex: '=',         fn: 'write',    desc: 'Equal'},
                     lower:  { label: '<',           latex: '<',         fn: 'write',    desc: 'Lower than'},
                     greater: { label: '>',          latex: '>',         fn: 'write',    desc: 'Greater than'},
-                    subscript: { label: 'x&#8345;', latex: '_',         fn: 'cmd',      desc: 'Subscript'},
-                    lbrace: { label: '{',           latex: '{',   fn: 'cmd',      desc: 'Left brace/curly bracket'},
-                    rbrace: { label: '}',           latex: '}',  fn: 'cmd',      desc: 'Right brace/curly bracket'},
-                    lparen: { label: '(',           latex: '(',   fn: 'cmd',      desc: 'Left parenthese/round bracket'},
-                    rparen: { label: ')',           latex: ')',  fn: 'cmd',      desc: 'Right parenthese/round bracket'},
+                    subscript: { label: '_',        latex: '_',         fn: 'cmd',      desc: 'Subscript'},
+                    lbrace: { label: '{',           latex: '{',         fn: 'cmd',      desc: 'Left brace/curly bracket'},
+                    rbrace: { label: '}',           latex: '}',         fn: 'cmd',      desc: 'Right brace/curly bracket'},
+                    lparen: { label: '(',           latex: '(',         fn: 'write',    desc: 'Left parenthese/round bracket'},
+                    rparen: { label: ')',           latex: ')',         fn: 'write',    desc: 'Right parenthese/round bracket'},
                     integral: { label: '&#x222b;',  latex: '\\int',     fn: 'cmd',      desc: 'Indefinitve integral'},
                     timesdot: { label: '·',         latex: '\\cdot',    fn: 'cmd',      desc: 'Times dot'},
+                    triangle: { label: '&#9651;',   latex: '\\triangle',fn: 'cmd',      desc: 'Triangle'},
+                    similar: { label: '&sim;',      latex: '\\sim',     fn: 'cmd',      desc: 'Similar'},
+                    paral:   { label: '&#8741;',    latex: '\\parallel',fn: 'cmd',      desc: 'Is parallel with'},
```

Comment:
The triangle and parallel buttons appear to render slightly bigger than the other symbols, creating gaps. Is there some place you can reduce the padding or the font size of these 2?